hail/python/test/hail/genetics/test_reference_genome.py

```diff
@@ -136,3 +136,31 @@ def test_liftover_strand(self):
             hl.eval(hl.liftover(hl.parse_locus_interval('1:10000-10000', reference_genome='GRCh37'), 'GRCh38'))
 
         grch37.remove_liftover("GRCh38")
+
+    def test_read_custom_reference_genome(self):
+        # this test doesn't behave properly if these reference genomes are already defined in scope.
```

Comment:
I used to have delete reference functionality to delete them at the end of a test. Not sure if that still is there.

Reply:
I poked around a little bit and didn't see anything. (I've tried to choose names that won't collide with other reference genomes; this assertion is just to catch if someone adds tests with references that have the same names in the future.)